spec/mailers/invitation_spec.rb

```diff
@@ -0,0 +1,5 @@
+require "rails_helper"
+
+RSpec.describe InvitationMailer, type: :mailer do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
I deleted this line in the other auto created email test file so there wouldn't be a skipped test